src/Synapse/Synapse/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Updated `New-AzSynapseSparkPool` and `Update-AzSynapseSparkPool` to support for setting spark pool configuration artifact by `-SparkCongifuration`, parameter `-SparkConfigFilePath` is deprecated now
```

Comment:
What do you mean _parameter `-SparkConfigFilePath` is deprecated now_? Does this parameter not work anymore? Will current change will break previous user experience?

Reply:
Specifying spark pool configuration using file path in Synapse Studio and Azure Portal is deprecated, now we can only make it through spark configuration artifact, so add a new parameter to support it.
The old parameter -SparkConfigFilePath is useless now, it won't take any effect for updating spark pool.

Reply:
So I guess in our next major release, SparkConfigFilePath should be removed from these cmdlets, am I correct? If so, please add upcoming breaking change message for this parameter

What happens if user uses SparkConfigFilePath in their current scripts? Will these scripts break now? If so, seems like it's a breaking change from service team.

Reply:
> I guess in our next major release, SparkConfigFilePath should be removed from these cmdlets, am I correct? If so, please add upcoming breaking change message for this parameter
>   yes, SparkConfigFilePath  should be removed in future.
> What happens if user uses SparkConfigFilePath in their current scripts? Will these scripts break now? If so, seems like it's a breaking change from service team.
>  if user specify -SparkConfigFilePath  parameter in their current scripts, they will not break.

Reply:
- Let's follow https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/breakingchange_release_process#how-to-releae-a-breaking-change to pre-announce the deprecation of SparkConfigFilePath
- Need to confirm when SparkConfigFilePath will not work for customer, it may still be regarded as a breaking change.

Reply:
> * Let's follow https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/breakingchange_release_process#how-to-releae-a-breaking-change to pre-announce the deprecation of SparkConfigFilePath
    Added the breaking change notifications for existing cmdlets parameter.

> * Need to confirm when SparkConfigFilePath will not work for customer, it may still be regarded as a breaking change.
    Confirmed, the SparkConfigFilePath may stop working until middle of this December.